content/documentation/5-feature-guides/1-streams/0-index.md

```diff
@@ -6,3 +6,123 @@ summary: true
 ---
 
 Discuss the set of features that will be covered.
```

Comment:
This line is causing the generated page to not render properly.  Needs to be moved below the meta-tags.